src/registry_schemas/example_data/schema_data.py

```diff
@@ -1210,6 +1210,11 @@
     'fileKey': '011e332d-1b8e-4218-8710-ad8ac1fbc592.pdf'
 }
 
+CONSENT_CONTINUATION_OUT = {
+    'orderDetails': 'A note to explain the consent to continue out',
+    'courtOrder': COURT_ORDER
```

Comment:
Where are we storing the Consent's expiry date?

Reply:
^^ The UI will not provide the expiry date -- the BE will compute it upon filing.

Reply:
Yeah! As we discussed I will create a new filed in business table. Thats not part of schema